python_modules/dagster-graphql/dagster_graphql/implementation/loader.py

```diff
@@ -333,6 +333,37 @@ def _fetch(self) -> None:
         }
 
 
+class CountByPartitionLoader:
```

Comment:
Can we sanity check this against a user w/ a large number of assets?

Reply:
Hmm @prha  I'm not sure how best to check this without shipping it to cloud, but we could feature-flag the client-side part of this and that'd let us check the performance maybe? I did put a client-side cap of 50 asset keys in any given request for counts, and we could make that lower while we evaluate the performance.

Is this something that shadow dagit could do?

Reply:
Yeah, definitely can do it with shadow dagit... You will have to rebuild dagit in internal using a virtual env with your oss branch checked out.